src/index.html

```diff
@@ -11,12 +11,63 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="logo">
+          <a href="#">
+            <img
+              src="/src/images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav">
+          <ul>
+            <li>
+              <a
+                href="#"
+                class="is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li><a href="#">SAMSUNG</a></li>
+            <li><a href="#">SMARTPHONES</a></li>
+            <li>
+              <a
+                href="#"
+                data-qa="hover"
+              >
+                LAPTOPS & COMPUTERS
+              </a>
+            </li>
+            <li><a href="#">GADGETS</a></li>
+            <li><a href="#">TABLETS</a></li>
```

Comment:
The text 'TABLETS' is hardcoded in uppercase. Consider using CSS to style the text to uppercase instead of hardcoding it in the HTML.